gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagementStateStore.java

```diff
@@ -77,22 +77,11 @@ public interface DagManagementStateStore {
 
   /**
    * This marks the dag as a failed one.
-   * Failed dags are queried using {@link DagManagementStateStore#getFailedDag(DagManager.DagId)} ()} later to be retried.
+   * Failed dags are queried using {@link DagManagementStateStore#getDag(DagManager.DagId)} ()} later to be retried.
```

Comment:
does it remain useful to both retrieve the DAG while also asserting that it's failed?